psutil/_psutil_posix.c

```diff
@@ -16,7 +16,7 @@
 #include <net/if.h>
 
 #ifdef PSUTIL_SUNOS10
-#include "arch/solaris10/ifaddrs.h"
+#include "arch/solaris/v10/ifaddrs.h"
```

Comment:
solaris x86 does not compile at all due to mib2.h problem. The thing is that there is not directory named solaris10 in psutil. The directory is solaris/v10/. Right?